core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -810,10 +810,8 @@ class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient, isSecure: Boo
   /**
    * Returns all reassignments.
    * @return the reassignments for each partition.
-   * @deprecated Use the PartitionReassignment Kafka API instead
    */
-  @Deprecated
```

Comment:
It doesn't make sense to deprecate a method in an internal class that is still needed by our code. Removed the deprecation of various `KafkaZkClient` classes. cc @stanislavkozlovski